<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondaryView</class>
 <widget class="QDialog" name="SecondaryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Var and Prog Parameters</string>
  </property>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Variables</string>
   </property>
   <widget class="QPushButton" name="pushButtonSupvar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAjvar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditvar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>571</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Programmes</string>
   </property>
   <widget class="QPushButton" name="pushButtonSupprof">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAjprog">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditprog">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>261</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonquit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>580</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRefresh">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Recharger</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
